hyperstart.go

```diff
@@ -296,6 +296,24 @@ func (h *hyper) bindMountContainerMounts(podID string, c Container, mounts []Mou
 	return fsmap, nil
 }
 
+func (h *hyper) bindUnmountContainerMounts(podID string, c *Container) error {
```

Comment:
The implementation for the bind-mounting is agent/hyperstart specific. That's why I put it here, along the same lines of bindUnmountContainerRootfs function that was defined for hyperstart